rero/rero-ils-ui
# search list: add extend link when no result has been found

* Adds a link to extend the search to all documents (public and admin).
* Adds a control on document brief view if the document exists.
* Changes boolean to new observable on adminMode configuration.
* Improves dynamic routing system.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Dependencies

* rero/ng-core#278

## How to test?

- Search for a term that does not exist in the results (Ex: Cheval)
- See if the extend link is present
- Click on the link and see if the url has changed and if you have results.

## Code review check list
